external-crates/move/crates/move-vm-profiler/src/lib.rs

```diff
@@ -372,9 +370,7 @@ macro_rules! gas_profiler_feature_enabled {
 #[macro_export]
 macro_rules! gas_profiler_feature_disabled {
     ($($tt:tt)*) => {
-        if !cfg!(feature = "gas-profiler") {
-            $($tt)*
-        }
+        $($tt)*
```

Comment:
Seems fishy that both macros are doing the same thing now

Reply:
Which ones?

Reply:
`gas_profiler_feature_enabled` and `gas_profiler_feature_disabled`

Reply:
They do the same thing but are gated by the feature differently so only one exists at at time

Reply:
The one without `feature = "gas-profiler"` is never used anymore

Reply:
Yeah, but should I remove it since it's in `external-crates?`